src/Datadog.Trace.ClrProfiler.Native/Datadog.Trace.ClrProfiler.Native.vcxproj

```diff
@@ -138,9 +138,10 @@
     <ClInclude Include="integration.h" />
     <ClInclude Include="integration_loader.h" />
     <ClInclude Include="IntegrationType.h" />
+    <ClInclude Include="iterators.h" />
```

Comment:
It looks like `iterators.h` wasn't added to the repo which is causing the [CI build](https://datadog-apm.visualstudio.com/dd-trace-csharp/default/_build/results?buildId=126&_a=summary&view=logs) to fail.